packages/roc-package-web-app-react/app/shared/flux/create-store.js

```diff
@@ -19,9 +19,10 @@ export default function createReduxStore(reducers, ...middlewares) {
     return (callback) =>
         (history, initialState) => {
             let finalCreateStore;
+            const normalMiddlewares = [].concat(middlewares);
```

Comment:
That would also work yes and when transpiled it will actually become what we have currently. So it all comes down to style preference. What you suggested might be a little "cleaner". 

Let's revisit this in the future.